MODULE.bazel

```diff
@@ -0,0 +1,10 @@
+module(
+    name = "jsoncpp",
+    version = "1.9.5",
+)
+
+bazel_dep(
+    name = "rules_cc",
```

Comment:
> There is no need to use rules from this repository just yet. If you want to use rules_cc anyway, add the following to your WORKSPACE file:

https://github.com/bazelbuild/rules_cc?tab=readme-ov-file#getting-started

What's that about? Sorry for my lack of background here but I don't want to depend on something that's perhaps not baked, or not something a library needs to mention.

Reply:
That’s also mine. You’re right that’s not necessary. 
Thank you for the clarification.
I removed it.